apollo-router/src/plugins/response_cache/postgres.rs

```diff
@@ -72,6 +73,11 @@ pub(crate) struct PostgresCacheConfig {
     /// Useful when running tests in parallel to avoid conflicts
     #[serde(default)]
     pub(crate) namespace: Option<String>,
+
+    #[serde(deserialize_with = "humantime_serde::deserialize", default)]
+    #[schemars(with = "Option<String>", default)]
+    /// Specifies the interval between cache cleanup operations (e.g., "2 hours", "30mins"). Default: 30mins
+    pub(crate) cleanup_interval: Option<Duration>,
```

Comment:
Won't this default to `None`? Where is the 30min coming from?

I assume we'll always want some cleanup interval, so would it be easier to have this be a `Duration` and use the serde default fn to specify 30 minutes?

Reply:
Also `humantime` **doesn't seem to support** `mins`, just `min`
```suggestion
    #[serde(deserialize_with = "humantime_serde::deserialize", default)]
    #[schemars(with = "Option<String>", default)]
    /// Specifies the interval between cache cleanup operations (e.g., "2 hours", "30min"). Default: 30min
    pub(crate) cleanup_interval: Option<Duration>,
```